api/v1/playbook_actions.go

```diff
@@ -98,6 +98,26 @@ type GithubAction struct {
 	Workflows []GithubWorkflow `yaml:"workflows,omitempty" json:"workflows,omitempty" template:"true"`
 }
 
+type AzureDevopsPipeline struct {
+	// ID is the pipeline ID
+	ID string `json:"id"`
+	// Version is the pipeline version
+	Version string `json:"version,omitempty" template:"true"`
+	// ApiVersion is the version of the API to use. Defaults to "7.1-preview.1"
+	ApiVersion string `json:"apiVersion,omitempty" template:"true"`
+}
+
+type AzureDevopsAction struct {
+	// Org is the name of the Azure DevOps organization
+	Org string `json:"org" template:"true"`
+	// Project ID or project name
+	Project string `json:"project" template:"true"`
+	// Token is the personal access token
+	Token types.EnvVar `yaml:"token" json:"token"`
+	// Pipelines is the list of azure pipelines to invoke
+	Pipelines []AzureDevopsPipeline `json:"pipelines,omitempty" template:"true"`
```

Comment:
I don't think we would execute multiple pipelines in 1 action, 1 action == 1 pipeline run so we can inline them here:

The action would take an id for pipeline id, branch and a map of parameters.